home/views/__init__.py

```diff
@@ -0,0 +1,39 @@
+"""Views for the home app."""
```

Comment:
Leave this file empty. Force the urls to import the full path to the views instead.